src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -130,66 +238,68 @@ public static SpeakerConfig calculateShooterSpeeds(double distance) {
-        bottomSpeed =
+                    * (speakerConfigList[i + 1].getSpeedLeftInRPM()
+                        - speakerConfigList[i].getSpeedLeftInRPM()))
+                + speakerConfigList[i].getSpeedLeftInRPM();
+        rightSpeed =
             (distanceProportion
-                    * (speakerConfigList[i + 1].getSpeedBottomInRPM()
-                        - speakerConfigList[i].getSpeedBottomInRPM()))
-                + speakerConfigList[i].getSpeedBottomInRPM();
-        speeds = new SpeakerConfig(distance, topSpeed, bottomSpeed);
-        System.out.println(i + " " + distance);
+                    * (speakerConfigList[i + 1].getSpeedRightInRPM()
+                        - speakerConfigList[i].getSpeedRightInRPM()))
+                + speakerConfigList[i].getSpeedRightInRPM();
+        speeds = new SpeakerConfig(distance, leftSpeed, rightSpeed);
         return speeds;
       }
     }
 
     // If distance above maximum, set speed to maximum.
-    topSpeed = speakerConfigList[speakerConfigList.length - 1].getSpeedTopInRPM();
-    bottomSpeed = speakerConfigList[speakerConfigList.length - 1].getSpeedBottomInRPM();
-    speeds = new SpeakerConfig(distance, topSpeed, bottomSpeed);
+    leftSpeed = speakerConfigList[speakerConfigList.length - 1].getSpeedLeftInRPM();
+    rightSpeed = speakerConfigList[speakerConfigList.length - 1].getSpeedRightInRPM();
+    speeds = new SpeakerConfig(distance, leftSpeed, rightSpeed);
     return speeds;
   }
 
   // Distance and speed in inches and RPM respectively.
   public static class SpeakerConfig {
     private double distanceInInches;
-    private double speedTopInRPM;
-    private double speedBottomInRPM;
+    private double speedLeftInRPM;
+    private double speedRightInRPM;
 
-    public SpeakerConfig(double distance, double speedTop, double speedBottom) {
+    public SpeakerConfig(double distance, double speedLeft, double speedRight) {
       this.distanceInInches = distance;
-      this.speedTopInRPM = speedTop;
-      this.speedBottomInRPM = speedBottom;
+      this.speedLeftInRPM = speedLeft;
+      this.speedRightInRPM = speedRight;
     }
 
     public double getDistanceInInches() {
       return distanceInInches;
     }
 
-    public double getSpeedTopInRPM() {
-      return speedTopInRPM;
+    // Motor RPM, NOT roller RPM
+    public double getSpeedLeftInRPM() {
+      return speedLeftInRPM;
     }
 
-    public double getSpeedBottomInRPM() {
-      return speedBottomInRPM;
+    // Motor RPM, NOT roller RPM
+    public double getSpeedRightInRPM() {
+      return speedRightInRPM;
     }
   }
 
+  // Motor RPM, NOT roller RPM
   private static SpeakerConfig[] speakerConfigList = {
-    new SpeakerConfig(0, 450, 250),
-    new SpeakerConfig(40, 550, 350),
-    new SpeakerConfig(195, 750, 500),
-    new SpeakerConfig(290, 1000, 700)
+    new SpeakerConfig(0, 250, 250),
+    new SpeakerConfig(40, 350, 350),
+    new SpeakerConfig(195, 500, 500),
+    new SpeakerConfig(290, 700, 700)
```

Comment:
We do want one side faster than the other, right?

Reply:
Maybe, but 1. I'm not sure which, and 2. The numbers will be reliant on testing anyway